Tasks/PackerBuild/DefaultTemplates/default.linux.template.json

```diff
@@ -0,0 +1,76 @@
+{
+  "variables": {
+    "client_id": "",
+    "client_secret": "",
+    "subscription_id": "",
+    "tenant_id": "",
+    "object_id": "",
+    "resource_group": "",
+    "storage_account": "",
+    "image_publisher": "",
+    "image_offer": "",
+    "image_sku": "",
+    "location": "",
+    "script_path": "",
+    "script_name": "",
+    "package_path": "",
+    "package_name": "",
+    "capture_name_prefix": ""
+  },
+  "builders": [{
+        "type": "azure-arm",
+        "client_id": "{{user `client_id`}}",
+        "client_secret": "{{user `client_secret`}}",
+        "resource_group_name": "{{user `resource_group`}}",
+        "storage_account": "{{user `storage_account`}}",
+        "subscription_id": "{{user `subscription_id`}}",
+        "tenant_id": "{{user `tenant_id`}}",
+        "object_id": "{{user `object_id`}}",
+                
+        "os_type": "Linux",
+        "image_publisher": "{{user `image_publisher`}}",
+        "image_offer": "{{user `image_offer`}}",
+        "image_sku": "{{user `image_sku`}}",
+        "location": "{{user `location`}}",
+        "vm_size": "Standard_D3_v2",
+
+        "capture_container_name": "vsts-buildimagetask",
+        "capture_name_prefix": "{{user `capture_name_prefix`}}"
+    }],
+    "provisioners": [{
+        "type": "shell",
+        "execute_command": "chmod +x {{ .Path }}; {{ .Vars }} sudo -E sh '{{ .Path }}'",
+        "inline": [
+            "rm -rf /deployTemp",
+            "mkdir /deployTemp",
+            "chmod 777 /deployTemp"
```

Comment:
setting mod here since the next step which is file upload will need this. Packer does not provide a way to tell file upload to be done with sudo(unlike shell provisioner). I had tried with 700, but did not work.